apps/assistants/sync.py

```diff
@@ -105,7 +105,7 @@ def push_assistant_to_openai(assistant: OpenAiAssistant, internal_tools: list |
     data["tool_resources"] = _sync_tool_resources(assistant)
 
     if internal_tools:
-        data["tools"] = [convert_to_openai_tool(tool) for tool in internal_tools]
+        data["tools"].extend([convert_to_openai_tool(tool) for tool in internal_tools])
```

Comment:
This bug has been around for ages and likely contributes to some of the weird assistant behaviour we've seen when tools are used in conjunction with builtin tools.